source/common/router/config_utility.h

```diff
@@ -50,14 +50,14 @@ class ConfigUtility {
   static Upstream::ResourcePriority parsePriority(const envoy::api::v2::RoutingPriority& priority);
 
   /**
-   * See if the specified headers are present in the request headers.
-   * @param headers supplies the list of headers to match
-   * @param request_headers supplies the list of request headers to compare against search_list
+   * See if the headers specified in the config are present in a request.
+   * @param request_headers supplies the headers from the request.
+   * @param request_headers supplies the list of configured header conditions on which to match.
    * @return true all the headers (and values) in the search_list set are found in the
```

Comment:
nit: `@return bool true if all the headers...` (it wasn't right before, but might as well fix it now!)

And I think it should say `config_headers` vector rather than search_list?